test/flows/am_i_getting_minimum_wage_flow_test.rb

```diff
@@ -0,0 +1,153 @@
+require "test_helper"
+require "support/flow_test_helper"
+require "support/shared_flows/minimum_wage_flow_test_helper"
+
+class AmIGettingMinimumWageFlowTest < ActiveSupport::TestCase
+  include FlowTestHelper
+  extend MinimumWageFlowTestHelper
+
+  setup do
+    testing_flow AmIGettingMinimumWageFlow
+    @living_wage_age = "25"
+  end
+
+  should "render a start page" do
+    assert_rendered_start_page
+  end
+
+  context "test shared questions" do
+    test_shared_minimum_wage_flow_questions
+  end
+
+  context "outcome: current_payment_above" do
+    setup do
+      testing_node :current_payment_above
+      add_responses what_would_you_like_to_check?: "current_payment",
+                    are_you_an_apprentice?: "not_an_apprentice",
+                    how_old_are_you?: "16",
+                    how_often_do_you_get_paid?: "15",
+                    how_many_hours_do_you_work?: "5",
+                    how_much_are_you_paid_during_pay_period?: "1000",
+                    is_provided_with_accommodation?: "yes_charged",
+                    current_accommodation_charge?: "1",
+                    current_accommodation_usage?: "5",
+                    does_employer_charge_for_job_requirements?: "no",
+                    current_additional_work_outside_shift?: "yes",
+                    current_paid_for_work_outside_shift?: "yes"
+    end
+
+    should "render living wage copy if a user is eligible for the national living wage" do
+      add_responses how_old_are_you?: @living_wage_age
+      assert_rendered_outcome text: "you appear to be getting the National Living Wage"
+    end
+
+    should "render minimum wage copy if a user is not eligible for the national living wage" do
```

Comment:
```suggestion
    should "render minimum wage copy if a user is not eligible for the national minimum wage" do
```

Reply:
The logic is actually based on the else clause of a living wage conditional. 

https://github.com/alphagov/smart-answers/blob/aa5425c0939f94f290463c7bcb65bc71242923ca/app/flows/am_i_getting_minimum_wage_flow/outcomes/current_payment_above.erb#L3-L7

Similar situation in the other cases you've flagged.